telosnetwork/telos-wallet
# Hotfix - modal toggle, exclusion list, incorrect network notification

# Fixes #401, #302, #436 

## Description

- upgrade web3modal version
- update syntax for modal and include exclusion list 
- fix bug where modal is permanently dismissed
- restores error message (disable loading indicators) on send if mobile wallet is on incorrect chain

## Test scenarios


1. click 'connect wallet'
2. click outside of modal, or close modal
3. click 'connect wallet' again
4. modal should re-appear 
5. should only show MetaMask option on Desktop, only recommended on mobile
6. select wrong chain in MM mobile, attempt to send transaction, should see error message indicating wrong network, switching to correct network should allow tx to succeed
7. send tokens (main net and test net)

## Checklist:
